packages/dbgpt-core/src/dbgpt/agent/skill/loader.py

```
@@ -113,12 +113,16 @@ def load_skills_from_directory(
-                ".yaml",
-                ".yml",
-                ".md",
-            ]:
+            if (
+                file_path.is_file()
+                and file_path.suffix
+                in [
+                    ".json",
+                    ".yaml",
+                    ".yml",
+                ]
```

Ruff (0.16.0)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)